### PR TITLE
HG-85: Loader / Progress bar / circular

### PR DESCRIPTION
**Pull request by AlexP-anec**

### CODE REVIEW

File: src/components/Loader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    | 91
+    | 92
+    | 93
+    | 94
+    | 95
+    | 96
+    | 97
+    | 98
+    | 99
+    | 100;
```
Comment (SergeySychevskiy) [inline src/components/Loader/index.tsx]: It's a pity that TS still doesn't have anything for such case, but I'd rather use just `value: number` and if it's bigger than `100` - we just show 100.
But in general the way you did it is also fine, just wanted to share my thoughts

File: src/components/Loader/index.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  flat?: boolean;
+  className?: string;
+}
+
+const TEST_ID = '@fv/Loader';
+
+export const Loader: React.FC<ILoaderProps> = ({ variant = 'circular', flat = false, progress = -1 }) => {
+  const calculateProgress = () => {
```
Comment (SergeySychevskiy) [inline src/components/Loader/index.tsx:20]: What's the point of `progress === -1`? If I understood correctly if progress === -1 it should infinite animation. Maybe we should show it if there are no progress value?

File: src/components/Loader/index.tsx (hunk 1 above)
Comment (AlexP-anec) [reply src/components/Loader/index.tsx]: Yeah, i researched a bit but no good solution. 

Let's see some other opinions and I will comply with the majority :)

File: src/components/Loader/index.tsx (hunk 2 above)
Comment (AlexP-anec) [reply src/components/Loader/index.tsx:20]: Yes, it was simpler for the storybook as well to just put -1 (the default value is -1 so you wouldn't have to add a progress prop -1 to show the loader).

**Review by SergeySychevskiy — DISMISSED** [review]

File: src/components/Loader/index.tsx (hunk 2 above)
Comment (AlexPilica) [reply src/components/Loader/index.tsx:20]: updated!

File: src/components/Loader/index.tsx (hunk 1 above)
Comment (AlexP-anec) [reply src/components/Loader/index.tsx]: updated!

**Review by Nagibaba — APPROVED** [review]

**Pull request merged**